lib/models/NotificationConfiguration.js

```diff
@@ -91,8 +91,8 @@ class NotificationConfiguration {
         }
         const queueConfig = notificationConfig.QueueConfiguration;
         if (!queueConfig || !queueConfig[0] || queueConfig[0] === '') {
```

Comment:
Not changed in this PR, but the last test is superfluous (covered by the 2nd, as an empty string is false-ish)